joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitor.scala

```diff
@@ -1678,7 +1688,8 @@ class PythonAstVisitor(
           xMayHaveSideEffects = !attribute.value.isInstanceOf[ast.Name],
           lineAndColOf(call),
           argumentNodes,
-          keywordArgNodes
+          keywordArgNodes,
+          Some(relFileName + ":<module>.")
```

Comment:
@DavidBakerEffendi 

This causes UT failures in `TypeRecoveryPassTests.scala` and so I have disabled its usage in the call flow. However, I used it since it did help me in resolving some fullnames. Can you tell me how this could be modified to make it more accurate?

Reply:
I will have a look now

Reply:
Removed these changes. I will do these changes in a separate PR so that the others can go in. Marking this comment resolved since changes have been moved out.